internal/pkg/agent/cmd/enroll.go

```diff
@@ -75,6 +75,7 @@ func addEnrollFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolP("delay-enroll", "", false, "Delays enrollment to occur on first start of the Elastic Agent service")
 	cmd.Flags().DurationP("daemon-timeout", "", 0, "Timeout waiting for Elastic Agent daemon")
 	cmd.Flags().DurationP("fleet-server-timeout", "", 0, "Timeout waiting for Fleet Server to be ready to start enrollment")
+	cmd.Flags().Bool("skip-daemon-reload", false, "Skip daemon reload after enrolling")
```

Comment:
If this flag is not intended for generic use (that is if a regular user is not supposed to specify it) maybe we want to hide it ?

Reply:
I agree with @pchila it is only being used by the container it should be hidden. I don't think we want users to use it and then complain that things didn't work as expected.